### PR TITLE
javascript1-week3/NicoQ

### PR DESCRIPTION
**Pull request by lnknfan90**
With the correct branch name 👍

### CODE REVIEW

File: 3_javascript/javascript1/week3/Item array removal/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (shpomp) [inline 3_javascript/javascript1/week3/Item array removal/index.html:1]: Neat file organization in the folders! 
Impressive systematic approach. 
There is a caveat, though, when trying to operate through your hw and f.x. run it with bash:
https://apple.stackexchange.com/questions/14683/how-to-cd-to-a-directory-with-a-name-containing-spaces-in-bash
Worth considering updating the naming convention 🙏🏽

File: 3_javascript/javascript1/week3/Series duration of my life/script.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      series.days * 24 * 60 + series.hours * 60 + series.minutes;
+    totalMinutes = totalMinutes + seriesMinutes;
+  }
+  for (let i = 0; i < seriesDurations.length; i++) {
+    const series = seriesDurations[i];
+    const seriesMinutes =
+      series.days * 24 * 60 + series.hours * 60 + series.minutes;
+    const percentage =
+      (seriesMinutes / (lifeSpanInYears * minutesInYear)) * 100;
+    console.log(`${series.title} took ${percentage.toFixed(4)}% of my life`);
```
Comment (shpomp) [inline 3_javascript/javascript1/week3/Series duration of my life/script.js:51]: Evidently, you are doing a lot of repetition here - same loop, some of the same actions. 
Optimize it to remove repetitive code, making sure you still get the same result 🙏🏽

**Review by shpomp — CHANGES_REQUESTED** [review]
Overall very nice job! 👍🏽 
Good logic, good structure. Some optimization needed on one of the tasks 🤓

File: 3_javascript/javascript1/week3/Item array removal/index.html (hunk 1 above)
Comment (lnknfan90) [reply 3_javascript/javascript1/week3/Item array removal/index.html:1]: Thanks! 
Regarding the naming, you are right, I had not thought about it, I will keep it in mind from now on :)

**Pull request closed without merging**